draft-ietf-tls-tls13.md

```diff
@@ -4319,6 +4322,31 @@ will receive various TLS 1.x versions in this field, however its value
 MUST always be ignored.
 
 
+## Zero-RTT backwards compatibility
```

Comment:
This should probably be a subsection under "Negotiating with an older server". (add another '#' and move this all to before "Negotiating with an older client") Also negates the need for the redundant language added to that section.